bot/pagination.py

```diff
@@ -339,16 +345,14 @@ def add_image(self, image: str = None) -> None:
                        prefix: str = "", suffix: str = "", timeout: int = 300,
                        exception_on_empty_embed: bool = False):
         """
-        Use a paginator and set of reactions to provide
-        pagination over a set of title/image pairs.The reactions are
-        used to switch page, or to finish with pagination.
+        Use a paginator and set of reactions to provide pagination over a set of title/image pairs.
 
-        When used, this will send a message using `ctx.send()` and
-        apply a set of reactions to it. These reactions may
-        be used to change page, or to remove pagination from the message.
+        The reactions are used to switch page, or to finish with pagination.
 
-        Note: Pagination will be removed automatically
-        if no reaction is added for five minutes (300 seconds).
+        When used, this will send a message using `ctx.send()` and apply a set of reactions to it.
```

Comment:
```suggestion
        When used, this will send a message using ctx.send() and apply a set of reactions to it.
```

Reply:
Same as before, prefer to keep wrapped in backticks.